content/syllabuses/short-course-devops-foundational.md

```diff
@@ -0,0 +1,80 @@
+---
+title: DevOps Short Course – Foundational
+---
+
+Welcome to the DevOps Short Course, the foundational level.
+
+This course is to be completed in 6 weeks.
+
+Learners should spend 4 to 6 hours every week on self paced learning, including reading, working on tasks, quizzes etc.
+
+## General information
+
+### Group session structure
+We will have two 1h group sessions per week. They will mostly follow the structure below:
+
+#### Session 1
+- Review tasks and questions from last week
+- Workshop: the facilitator will present concepts, explain the content, ask questions, give answers, open a dialogue with the learners, demonstrate practical concepts
```

Comment:
I understand it does not follow the practices from the Gitbook, I wrote this before I read that content.

I opened this discussion on [Slack 9 days ago](https://umuzihq.slack.com/archives/C06PP6PQV6Z/p1710937150974649) and got no answer. I chatted with @t-witchy the beginning of this week and the session structure was approved by her.

I won't change this now.

What I'll do is:
- Try this structure for the first two modules, see how it goes, get feedback from the Umuzi team and understand if it works as a different session type, _an experiment_
- If it works, we can document and make it available as a new session type, if not, I'll write the following modules using the current available session types

Reply:
I'll have to follow up with @t-witchy. In general it's good to know that Whitney is not the person who defines or approves our pedagogical approach. That is my job.  She will spot certain kinds of issues and have some advice, but her approval does not mean it is approved. I still need to do the final check. There are good reasons for this.

I created and shared the docs on 14th of March I think. It was ages ago.  It states that lectures should be avoided. I have also spoken to Whitney about why we should avoid lectures so it is really strange that she approved. But maybe there is a misunderstanding or miscommunication. 

Whitney is your first line of support so I have been hanging back from doing that job. There are some communication glitches that seem to be cropping up as a result. Sorry about that. I'll chat to Whitney and see if we can figure out a better way to support SMEs. There should be a more explicit sign-off and feedback process.

It is ok to experiment but we do need to avoid things that we know are worth avoiding. Like lectures.  So presenting and explaining the content isn't something I approve of as a first step - the syllabus content should have presented and explained the content.  If a learner as a question or if the content falls short at getting an idea across then explaining is necessary. But it shouldn't be the first step.  

I recommend getting the learners to get through the content on their own, ahead of the class. Then focusing on interactions.  Instead of thinking about explicitly teaching the learners, think about setting them up to learn well. 

I understand that there is not a lot of time left before this course starts so it will be a challenge to adapt right now. But please do your best.

Reply:
Got it all.

I worked on the session structure to be a mix between Hotseat and Conceptual Question.

I'll give the learners some time on the beginning of every session to bring questions/opinions etc about the week's assignments and then I'll break the room and give them a conceptual question or problem to solve.

I'll explore this and see if there's enough time to do both during a 1h session. If not, I'll keep one session specifically for Hotseat and another for Conceptual.